### PR TITLE
feat: Ask for customer email on license items

### PR DESCRIPTION
**Pull request by nanu-c**
# Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Description

This adds the feature to ask for the vendor email, when there is a license item in the basket.
Screen Shot 2023-12-12 at 13 56 06

It also cleans up and fixes the problem, that  payment order requests have been posted always twice with `paymentStore.postOrder`

# Checklist:

- [ ] I have commented my code (or ChatGPT did), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings, neither in my IDE nor in my browser
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/locales/de.json [hunk 1]
```diff
@@ -124,5 +124,11 @@
   "errorText": "Upsi, es ist ein Fehler aufgetreten. Bitte versuche es erneut.",
 
   "export": "Exportieren",
-  "Logout": "Logout"
+  "Logout": "Logout",
+  "Save": "Speichern",
+  "No, cancel": "Nein, abbrechen",
```
Comment (lebe1) [inline src/locales/de.json:129]: Maybe we just use cancel here or "Abbrechen"?

**Review by lebe1 — COMMENTED** [review]
I love the feature that it saves the mail in a second run and shows it below the purchased items!!
Only thing missing is a translation when I chose to read everything in English. Besides that I could not find any gaps like a wrong mail input and everything works smooth :)

image

File: src/locales/de.json (hunk 1 above)
Comment (nanu-c) [reply src/locales/de.json:129]: Actually it's no exactly cancel, but more a oh i don't wanna give my email, so lets back

**Comment by nanu-c** [comment]
> I love the feature that it saves the mail in a second run and shows it below the purchased items!! Only thing missing is a translation when I chose to read everything in English. Besides that I could not find any gaps like a wrong mail input and everything works smooth :)
> 
I have fixed the english translation now, so i think we are good to go.

**Review by lebe1 — APPROVED** [review]
Lastly, I've added the missing translation item 'Back'. In case, you don't like the wording "Zurück" or "Back", you can change it of course!

**Pull request merged**